tests/integration/telemetry/api/accesslogs_test.go

```diff
@@ -138,10 +159,22 @@ spec:
 	t.ConfigIstio().YAML(common.GetAppNamespace().Name(), config).DeleteOrFail(t)
 }
 
-func runAccessLogsTests(t framework.TestContext, expectLogs bool) {
+func runAccessLogsTests(t framework.TestContext, expectLogs bool, hasTargetRef bool) {
 	testID := rand.String(16)
-	to := common.GetTarget()
+	to := common.GetTarget().(echo.Instances)
```

Comment:
You can't use this as the destination service for targetRef tests since the destination needs to be the gateway, not the backend service